proto/message.py

```diff
@@ -341,7 +345,9 @@ def from_json(cls, payload) -> "Message":
             ~.Message: An instance of the message class against which this
             method was called.
         """
-        return cls(Parse(payload, cls()._pb))
+        instance = cls()
+        Parse(payload, instance._pb)
+        return instance
```

Comment:
I'm having trouble wrapping my head around this - does calling `Parse` modify the instance that's passed in?

Reply:
Never mind! I read the [docs](https://googleapis.dev/python/protobuf/latest/google/protobuf/json_format.html#google.protobuf.json_format.Parse) on this and it makes sense now 🙃